modules/nextflow/src/main/groovy/nextflow/k8s/K8sDriverLauncher.groovy

```diff
@@ -137,24 +139,24 @@ class K8sDriverLauncher {
         this.k8sClient = makeK8sClient(k8sConfig)
         this.k8sConfig.checkStorageAndPaths(k8sClient)
         createK8sConfigMap()
-        createK8sLauncherPod()
-        waitPodStart()
+        createK8sLauncherJob()
+        waitJobStart()
```

Comment:
Can we keep the *Pod* naming, since pod is still the default resource?

Reply:
Well, shoud not be using job the target status? We can refactor back to pod, but once jobs will be default, we shall refactor back to job?

Reply:
> but once jobs will be default, we shall refactor back to job

Possibly, or even better in the future the pod ones will be deprecated and a new version with `Job` in the name it will be added. 

When doing this kind of change it's always important to not break compatibility with projects that can rely on the nextflow codebase.

Reply:
so ok, refactored to `Pod`